app/views/publisher_information/beta_capabilities.govspeak.md

```diff
@@ -1,11 +1,13 @@
-<span class="govuk-hint">Last updated: 9 November 2018</span>
+<span class="govuk-hint">Last updated: 1 February 2019</span>
 The Content Publisher beta tool can be used to publish the following document types:
 
 * News stories
 * Press releases
 
 Content created in Content Publisher cannot be edited in Whitehall publisher. Whitehall content cannot be edited in Content Publisher.
 
+Content created in Content Publisher can be featured on organisation homepages in Whitehall using the option to Feature non-GOV.UK government link.
```

Comment:
Should Feature non-GOV.UK government link be inside quotes? the capital F jarred for me a bit.

Reply:
Yes I agree. Can you add single quote - 'Feature non-GOV.UK government link' 

I'd made a note to check patterns for mentioning interface labels or markdown snippets in our in-app guidance - I think our tech writers do it differently to Whitehall docs. But not worth following up for now.